exchanges/lbank/lbank_wrapper.go

```diff
@@ -0,0 +1,523 @@
+	}
+
+	for key := range orderIDs {
+		if key != orders.CurrencyPair.String() {
+			continue
+		}
+		var x, y = 0, 0
+		var input string
+		var tempSlice []string
+		for x <= len(orderIDs[key]) {
+			x++
+			for y != x {
+				tempSlice = append(tempSlice, orderIDs[key][y])
+				if y%3 == 0 {
+					input = strings.Join(tempSlice, ",")
+					CancelResponse, err2 := l.RemoveOrder(key, input)
+					if err2 != nil {
+						return resp, err2
+					}
+					tempStringSuccess := strings.Split(CancelResponse.Success, ",")
+					for k := range tempStringSuccess {
+						resp.OrderStatus[tempStringSuccess[k]] = "Cancelled"
+					}
+					tempStringError := strings.Split(CancelResponse.Err, ",")
+					for l := range tempStringError {
+						resp.OrderStatus[tempStringError[l]] = "Failed"
+					}
+					tempSlice = tempSlice[:0]
+					y++
+				}
+				y++
+			}
+			input = strings.Join(tempSlice, ",")
+			CancelResponse, err2 := l.RemoveOrder(key, input)
+			if err2 != nil {
+				return resp, err2
+			}
+			tempStringSuccess := strings.Split(CancelResponse.Success, ",")
+			for k := range tempStringSuccess {
+				resp.OrderStatus[tempStringSuccess[k]] = "Cancelled"
+			}
+			tempStringError := strings.Split(CancelResponse.Err, ",")
+			for l := range tempStringError {
+				resp.OrderStatus[tempStringError[l]] = "Failed"
+			}
+			tempSlice = tempSlice[:0]
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderInfo returns information on a current open order
+func (l *Lbank) GetOrderInfo(orderID string) (exchange.OrderDetail, error) {
+	var resp exchange.OrderDetail
+	orderIDs, err := l.GetAllOpenOrderID()
+	if err != nil {
+		return resp, err
+	}
+
+	for key, val := range orderIDs {
+		for i := range val {
+			if val[i] != orderID {
+				continue
+			}
+			tempResp, err := l.QueryOrder(key, orderID)
+			if err != nil {
+				return resp, err
+			}
+			resp.Exchange = l.GetName()
+			resp.CurrencyPair = currency.NewPairFromString(key)
+			if strings.EqualFold(tempResp.Orders[0].Type, "buy") {
+				resp.OrderSide = exchange.BuyOrderSide
+			} else {
+				resp.OrderSide = exchange.SellOrderSide
+			}
+			z := tempResp.Orders[0].Status
+			switch {
+			case z == -1:
+				resp.Status = "cancelled"
+			case z == 1:
+				resp.Status = "on trading"
+			case z == 2:
+				resp.Status = "filled partially"
+			case z == 3:
+				resp.Status = "Filled totally"
+			case z == 4:
+				resp.Status = "Cancelling"
+			default:
+				return resp, fmt.Errorf("invalid order status: %v", tempResp.Orders[0].Status)
+			}
+			resp.Price = tempResp.Orders[0].Price
+			resp.Amount = tempResp.Orders[0].Amount
+			resp.ExecutedAmount = tempResp.Orders[0].DealAmount
+			resp.RemainingAmount = tempResp.Orders[0].Price - tempResp.Orders[0].DealAmount
+			resp.Fee, err = l.GetFeeByType(&exchange.FeeBuilder{
+				FeeType:       exchange.CryptocurrencyTradeFee,
+				Amount:        tempResp.Orders[0].Amount,
+				PurchasePrice: tempResp.Orders[0].Price})
+			if err != nil {
+				return resp, err
```

Comment:
@thrasher- 
In cases like this what is your preferred response? if the fee for whatever reason is invalid is it worth exiting the entire function or just set it to some sum such as -0.0 
(this also goes for above when we do it with multiple orders)